internal/cmd/build.go

```diff
@@ -114,7 +110,7 @@ func (b *Build) BuildFromSource(ctx context.Context, srcPath string, opts ...Bui
 	return nil
 }
 
-func (b *Build) validatePackage(pkg *packagecontent.Package) error {
+func (b *Build) ValidatePackage(_ context.Context, pkg *packagecontent.Package) error {
```

Comment:
Does this need the context.Context in the signature

Reply:
Yes. It's needed to match the `Validator` interface.